tests/unit/lifecycle/test_lifecycle.py

```diff
@@ -607,17 +607,16 @@ def fake_uname(cmd, *args, **kwargs):
         check_output_patcher.start()
         self.addCleanup(check_output_patcher.stop)
 
-        original_check_call = subprocess.check_call
-
-        def _fake_dpkg_deb(command, *args, **kwargs):
-            if "dpkg-deb" not in command:
-                return original_check_call(command, *args, **kwargs)
+        self.mock_extract_deb_name_version = fixtures.MockPatch(
```

Comment:
thought we agreed to call these fake_<foo>, in any case, you do not need to carry this in `self` if not going to be accessed in the tests, same for `self.mock_extract_deb` -> `fake_extract_deb`

Reply:
Agreed, Fixed!